cups/md5passwd.c

```diff
@@ -19,6 +19,8 @@
 /*
  * 'httpMD5()' - Compute the MD5 sum of the username:group:password.
  *
+ * Stubbed out in 2.4.0.
```

Comment:
Since this is part of the generated documentation, we should be a little more verbose here.

Something like: "This function used to compute the MD5 sum of the username:group:password tuple for HTTP Digest authentication. As of CUPS 2.4.0, it produces an empty string and you must instead use the @link cupsDoAuthentication@ or @link cupsHashData@ functions.